### PR TITLE
Improve type safety

### PR DESCRIPTION
**Pull request by shartrooper**

### CODE REVIEW

**Comment by shartrooper** [comment]
I've been checking out all the webhook event types available to create: member.created, member.deleted, invitation.created, invitation.deleted. If you add then in your own dashboard those work , which webhooks are missing?
https://user-images.githubusercontent.com/21326996/268274364-9afaf377-7543-4fb6-baf1-991cb2972116.png

**Comment by devkiran** [comment]
@shartrooper is this PR ready to review?

**Comment by shartrooper** [comment]
> @shartrooper is this PR ready to review?

Yes,  i marked it as draft until the webhooks matter was pending, since now is settled, the PR is ready to merge.

File: lib/retraced.ts [hunk 1]
```diff
@@ -83,7 +83,7 @@ export const getViewerToken = async (groupId: string, actorId: string) => {
 
   try {
     return await retracedClient.getViewerToken(groupId, actorId, true);
-  } catch (error: any) {
+  } catch (_error) {
```
Comment (devkiran) [inline lib/retraced.ts:86]: is this necessary?

**Review by devkiran — APPROVED** [review]
LGTM

**Pull request merged**